_maps/serendipity.json

```diff
@@ -0,0 +1,33 @@
+{
+    "map_name": "DLV Serendipity",
+    "map_link": "Serendipity",
+    "map_path": "map_files/Serendipity",
+    "map_file": ["Serendipity2.dmm", "Serendipity1.dmm"],
+	"traits": [{"Up": 1, "Linkage": "Self"}, {"Down": -1, "Linkage": "Self"}],
+    "space_ruin_levels": -1,
+    "space_empty_levels": 0,
+    "shuttles": {
+		"arrival": "arrival_serendipity",
+        "cargo": "cargo_serendipity",
+        "ferry": "ferry_fancy",
+        "emergency": "emergency_celerity"
+	},
+    "mine_disable": 1,
+    "ship_type": "/obj/structure/overmap/nanotrasen/serendipity/starter",
+	"starmap_path": "config/starmap/starmap.json",
+
+    "map_description": "Serendipity vessels are small, light, and fast. Due to a recent amendment to The Dominion of Light’s involvement in the Federation, they now manufacture and sell Serendipity ships to WhiteRapids directly. Originally used for scientific and medical research, these ships are now outfitted with a light array of weapons and all the equipment necessary for QRF.",
+    "manufacturer": "Dominion of Light Exploration and Development Council",
+	"mapper": "BetaCentari",
+    "commission_date": "Nov 2, 2262",
+    "strengths": ["Supremely Fast, Agile, and Small", "Robust Medical and Science Departments", "Set and Forget Reactor"],
+    "weaknesses": ["Fragile, Vulnerable to Breaches", "", "Little to No Privacy", "Gravity Generator Off At Roundstart"],
```

Comment:
Fragile meaning it has low armour, vulnerable to breaches meaning central chambers are large and will depressurize but since that wasn't evident from the get go I'll change it.